sympy/plotting/plot.py

```diff
@@ -67,14 +67,21 @@ def _arity(f):
        return len([p for p in param if p.kind == p.POSITIONAL_OR_KEYWORD])
 
 
+plot_backends = {}
+
+
 class Plot(object):
     """The central class of the plotting module.
 
     For interactive work the function ``plot`` is better suited.
 
     This class permits the plotting of sympy expressions using numerous
     backends (matplotlib, textplot, the old pyglet module for sympy, Google
-    charts api, etc).
+    charts api, etc).To choose between a backend use ``backend`` option while
```

Comment:
Well, I think it could be enough to say 'To choose a backend', but, of course, this is not my native language. I hope someone else would confirm this.